src/blog/_posts/2018-04-01-opera-52.md

```diff
@@ -0,0 +1,155 @@
+                        Math.sin((x + y) / 6)) * size / 4 + size / 12;
+                    const opening = Math.random() * 90;
+                    this.circle_arc(ctx, (x + 0.5) * size, (y + 0.5) * size,
+                                    circle_size, opening);
+                }
+            }
+        }
+    }
+
+    // Register our class under a specific name
+    registerPaint('circlearcpainter', CircleArcPainter);
+
+
+## Server Timing API
+
+With the new [Server Timing API](https://w3c.github.io/server-timing/)
+there is a well defined way for servers pass performance information
+to the browser through HTTP headers. With this information web pages
+can make even better informed decisions.
+
+In Developer Tools this information is displayed in the Network view
+by selecting the resource and activating the Timing tab.
+
+## CSS
+* The
+  [`:any-link`](https://developer.mozilla.org/en-US/docs/Web/CSS/:any-link)
+  pseudo selector can be used to style both visited and non-visited
+  links at the same time.  For colors, the `rgb` and `hsl` functions
+  now take an [optional fourth
+  alpha](https://drafts.csswg.org/css-color/#rgb-functions) value,
+  making them identical to `rgba` and `hsla`.
+* [`display:
+  contents`](https://drafts.csswg.org/css-display/#box-generation)
+  allows an element to wrap other elements without creating a box for
+  itself.
+
+## DOM
+* There is now an
+  [`assignedElements()`](https://html.spec.whatwg.org/#dom-slot-assignedelements)
+  on `<slot>` elements, which acts
+  like `assignedNodes()` but only returns element nodes.
+* [`HTMLAnchorElement.relList`](https://html.spec.whatwg.org/#dom-link-rellist)
+  will now give an accurate mapping of the `rel` attribute.
+* [`document.all`](https://html.spec.whatwg.org/#htmlallcollection)
+  can no longer be overwritten.
+
+## Feature Policy
+* A new [feature policy](https://wicg.github.io/feature-policy/)
+  `sync-xhr` willallow a site to block synchronous
+  XMLHttpRequests. Synchronous XMLHttpRequests block scripts until a
+  server has responded which can make them harmful for the
+  experience. If a site embeds untrusted, possibly malicious, content
+  (think ads), this adds a tool to lock that content down a bit.
+
+## Network
+* The https code is updated to match the [draft-23
+  version](https://tools.ietf.org/html/draft-ietf-tls-tls13-23) of the
+  TLS protocol.
+* [`Request.destination`](https://fetch.spec.whatwg.org/#concept-request-destination)
+  is added to give service workers a better understand on why a resource
+  is fetched.
+
+## Performance APIs
+* A `toJSON()` method has been added to `PerformanceResourceTiming`,
+  `PerformanceLongTaskTiming` and `TaskAttributionTiming`.
+
+## Security
+* The [download](https://html.spec.whatwg.org/#attr-hyperlink-download)
+  attribute will be ignored on anchor elements with
+  cross-origin attributes.
+* Certificates from Symantec's Legacy PKI issued after 1 December,
```

Comment:
I think dates are commonly formatted "1 December 2017" or "December 1, 2017". Former commonly on this side of the Atlantic ocean and latter on the other side, apparently. If you had used the latter here, I probably wouldn't have thought about it. With this spelling, I ended up reading Wikipedia articles about it. I'm pretty sure the world will end if we get it wrong.